technology-considerations/Considerations for authentication and authorization technology

```diff
@@ -0,0 +1,25 @@
+| Total   cost of ownership long term       | Entire risk & dev time                                           | 1     | Shared risk & dev                                            | 3     | Shared risk & dev                                              | 3     | Shared risk & dev                                                                                                                       | 3     |
+| Standard   Tech components                | Some                                                             | 3     | Some                                                         | 3     | Some                                                           | 3     | Some                                                                                                                                    | 3     |
+| Other   dept usage                        | Own bespoke build                                                | 1     | Digi services in use with   limitations                      | 2     | Not aware of use                                               | 3     | This is also used in other gov   depts so I've raised Auth0 to 3 and set keycloak to be the same                                        | 3     |
+| Failure   impact of component             | Risks, but own build to fix                                      | 2     | Risks, but more known                                        | 2     | Risks, unknown                                                 | 1     | We have full control over the   upgrade cycle of the software and the hosting. They are our risks but we have   full control over them. | 3     |
+| Security   risk mitigation                | Unknown, but should be able to   build with time                 | 3     | Known token issue threat                                     | 4     | Seems most secure                                              | 5     | As secure as auth0, it is an   OIDC provider                                                                                            | 4     |
+| Supports   GDPR                           | We need work to prove                                            | 3     | Established                                                  | 5     | Established                                                    | 5     | We need work to prove                                                                                                                   | 3     |
+| Supports   GDS                            | On balance                                                       | 3     | On balance                                                   | 3     | On balance                                                     | 3     | On balance                                                                                                                              | 3     |
```

Comment:
Did you find the GDS article you were using for this part of the evaluation? It would be worth adding a link. Or if "supports GDS" just means "is open source", maybe we should reword it to that?

Reply:
No, GDS have revised this and they talk about the total cost of ownership, rather than just recommending open source - I included the link.